.codecov.yml

```diff
@@ -14,11 +11,11 @@ coverage:
   status:
     changes: false
     patch:
-      default:
-        threshold: 0.01
-    project:
-      default:
-        threshold: 0.01
+      only_pulls: true
+      paths:
```

Comment:
I'm guessing this means it only ends up being run for those directories, right? If so, that's good that it avoids being run on, e.g., `Doc/*` changes, but shouldn't it also run on directories like `Objects/`?